### PR TITLE
notification toast component

### PR DESCRIPTION
**Pull request by ideaoforder**

### CODE REVIEW

File: scss/whiplash-ui.scss [hunk 1]
```diff
@@ -9,6 +9,7 @@
 @import "components/button";
 @import "components/breadcrumb";
 @import "components/loader";
+@import "components/notifications";
```
Comment (jacobroufa) [inline scss/whiplash-ui.scss]: I don't see the new file `/scss/components/_notifications.scss` in this commit.

Comment (ideaoforder) [inline scss/whiplash-ui.scss]: good point

File: scss/components/_notifications.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
+.notification-toast {
+  background: -webkit-linear-gradient(left, #e9faf6 1%, #deeaea);
```
Comment (jacobroufa) [inline scss/components/_notifications.scss]: Doesn't the preprocessor have a prefixer to handle these things?

File: scss/components/_notifications.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  background: linear-gradient(90deg, #e9faf6 1%, #deeaea);
+  // filter: progid: DXImageTransform.Microsoft.gradient(startColorstr="#e9faf6", endColorstr="#c1cccb", GradientType=1);
```
Comment (jacobroufa) [inline scss/components/_notifications.scss]: See L2 comment.

File: scss/components/_notifications.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  width: 100%;
+  color: #00bc93;
+  font-family: Lato, Aller-Light, Helvetica, Arial, sans-serif;
+  font-size: .875rem;
+  letter-spacing: .04rem;
+  -webkit-font-smoothing: antialiased;
+
+  &.pending {
+    background: #eae8ff;
+    background: -webkit-linear-gradient(left, #eae8ff, #bde9ed);
```
Comment (jacobroufa) [inline scss/components/_notifications.scss]: See L2 comment.

File: scss/components/_notifications.scss [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    background: linear-gradient(90deg, #eae8ff 0, #bde9ed);
+    color: #4800b6;
+  }
+
+  &.error {
+    background: -webkit-linear-gradient(left, rgba(234, 82, 94, .27), rgba(165, 34, 34, .39));
```
Comment (jacobroufa) [inline scss/components/_notifications.scss]: See L2 comment.

**Comment by jacobroufa** [comment]
I should add this https://github.com/postcss/autoprefixer if you were not sure what I meant. If you'd like me to add this, and have a set of required browsers to support, please let me know and I'll submit a PR.

**Comment by ideaoforder** [comment]
That'd be great @jacobroufa. The PR's all yours.

(I was just copying code from the UI lib site--needed notifications, and needed them in like 5 min, so this was the fast version. This is just pulled from docs--will remove when this is published.)

**Comment by jacobroufa** [comment]
Cheers. Then if you make the modifications to this PR to remove prefixes, I'm OK with it landing.

As that functionality is concerned, would you rather have all the prefixing done from within this repo, or would you like the autoprefixer set as a Rails plugin on `whiplash-docs`? Either way is simple enough to take care of.

**Comment by jacobroufa** [comment]
@ideaoforder check out #16 for the autoprefixing goodness. Land either of the PRs first, but if you decide not to clean up the prefixes in the `_notifications.scss` here, I'll just roll through the whole repo and clean up that sort of thing after both of these PRs are landed. It should be done anyway, so you can probably just merge this PR as is.

**Pull request merged**